docs/changes.rst

```diff
@@ -23,6 +23,14 @@ information to the contrary.
 
 
 ------------------
+3.8.4 - 2017-05-15
+------------------
+
+This is a compatibility release:  Hypothesis no longer imports ``sqlite3``
+before a SQLite database is first used, so the default settings now work on
+Python implementations compiled without sqlite support (eg BSD).
```

Comment:
I refuse to make this a blocking comment and am going to approve the change, but my preferred convention here is "e.g. " rather than "eg" and it would make me happy if you followed. :-)